inc/views/partials/post_meta.php

```diff
@@ -198,11 +210,33 @@ public static function get_time_tags() {
 		$created  = get_the_time( 'U' );
 		$format   = get_option( 'date_format' );
 		$modified = get_the_modified_time( 'U' );
-		$time     = '<time class="entry-date published" datetime="' . esc_attr( date_i18n( 'c', $created ) ) . '" content="' . esc_attr( date_i18n( 'Y-m-d', $created ) ) . '">' . esc_html( date_i18n( $format, $created ) ) . '</time>';
+
+		$prefixes = array(
+			'published' => '',
+			'updated'   => '',
+		);
+
+		/**
+		 * Filters the prefix of the published date and the updated date of a post.
+		 *
+		 * @param array $prefixes {
+		 *     Prefix location and value.
+		 *
+		 *     @type string $published The prefix for the published date.
+		 *     @type string $updated The prefix for the updated date.
+		 * }
+		 *
+		 * @since 2.11
+		 */
+		$prefixes         = apply_filters( 'neve_meta_date_prefix', $prefixes );
+		$published_prefix = array_key_exists( 'published', $prefixes ) ? $prefixes['published'] : '';
```

Comment:
I would check if the prefixes exist and if they are not empty, and then run kses here vs on content, where we will run even if there is an empty string.